config.go

```diff
@@ -41,3 +43,25 @@ type Configuration struct {
 	URL     string
 	Timeout time.Duration
 }
+
+// UnmarshalYAML parses a reporter configuration from YAML.
+func (configuration *Configuration) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	type rawConfiguration struct {
+		URL     string
+		Timeout uint64
+	}
+	// this is the default configuration
+	rawConfig := rawConfiguration{
+		URL:     "https://kroki.io",
```

Comment:
it should be `https://demo.kroki.io`:

```suggestion
		URL:     "https://demo.kroki.io",
```

Reply:
Updated